src/ui/include/mainwindow.h

```diff
@@ -207,9 +206,6 @@ private slots:
     QMenu*                m_tabContextMenu;
     QList<QAction *>      m_tabContextMenuActions;
     QLabel*               m_statusBar_fileFormat;
```

Comment:
These remaining `QLabel*`s should probably all just be `ClickableLabel*`s instead. I see no reason to have to up/down-cast them.

Reply:
I agree with this.